cloud/src/meta-service/meta_service.cpp

```diff
@@ -1923,11 +1925,37 @@ void MetaServiceImpl::get_delete_bitmap(google::protobuf::RpcController* control
         std::unique_ptr<RangeGetIterator> it;
         int64_t last_ver = -1;
         int64_t last_seg_id = -1;
+        int64_t round = 0;
         do {
-            err = txn->get(start_key, end_key, &it);
+#ifdef UNIT_TEST
```

Comment:
use syncpoint instead of MACRO, or change config::max_get_delete_bitmap_retry_times before testing
```
int n = 10000;
syncpoint(&n);
err = txn->get(start_key, end_key, &it, true, n);
```